Starting Klippy...
Args: ['/home/biqu/klipper/klippy/klippy.py', '/home/biqu/printer_data/config/printer.cfg', '-I', '/home/biqu/printer_data/comms/klippy.serial', '-l', '/home/biqu/printer_data/logs/klippy.log', '-a', '/home/biqu/printer_data/comms/klippy.sock']
Git version: 'v0.12.0-125-gbfb71bc2'
Branch: master
Remote: origin
Tracked URL: https://github.com/Klipper3d/klipper
CPU: 4 core ?
Python: '3.9.2 (default, Feb 28 2021, 17:03:44) \n[GCC 10.2.1 20210110]'
Start printer at Tue Mar 19 09:13:16 2024 (1710839596.9 83.4)
Config error
Traceback (most recent call last):
  File "/home/biqu/klipper/klippy/klippy.py", line 175, in _connect
    self._read_config()
  File "/home/biqu/klipper/klippy/klippy.py", line 134, in _read_config
    config = pconfig.read_main_config()
  File "/home/biqu/klipper/klippy/configfile.py", line 286, in read_main_config
    regular_config = self._build_config_wrapper(regular_data, filename)
  File "/home/biqu/klipper/klippy/configfile.py", line 273, in _build_config_wrapper
    self._parse_config(data, filename, fileconfig, set())
  File "/home/biqu/klipper/klippy/configfile.py", line 261, in _parse_config
    self._resolve_include(filename, include_spec, fileconfig,
  File "/home/biqu/klipper/klippy/configfile.py", line 234, in _resolve_include
    raise error("Include file '%s' does not exist" % (include_glob,))
configparser.Error: Include file '/home/biqu/printer_data/config/generic-bigtreetech-xxx.cfg' does not exist
webhooks client 281473184929392: New connection
webhooks client 281473184929392: Client info {'program': 'Moonraker', 'version': 'v0.8.0-324-ga3e4dac'}
Starting Klippy...
Args: ['/home/biqu/klipper/klippy/klippy.py', '/home/biqu/printer_data/config/printer.cfg', '-I', '/home/biqu/printer_data/comms/klippy.serial', '-l', '/home/biqu/printer_data/logs/klippy.log', '-a', '/home/biqu/printer_data/comms/klippy.sock']
Git version: 'v0.12.0-125-gbfb71bc2'
Branch: master
Remote: origin
Tracked URL: https://github.com/Klipper3d/klipper
CPU: 4 core ?
Python: '3.9.2 (default, Feb 28 2021, 17:03:44) \n[GCC 10.2.1 20210110]'
Start printer at Tue Mar 19 09:17:11 2024 (1710839831.2 17.1)
Config error
Traceback (most recent call last):
  File "/home/biqu/klipper/klippy/klippy.py", line 175, in _connect
    self._read_config()
  File "/home/biqu/klipper/klippy/klippy.py", line 134, in _read_config
    config = pconfig.read_main_config()
  File "/home/biqu/klipper/klippy/configfile.py", line 286, in read_main_config
    regular_config = self._build_config_wrapper(regular_data, filename)
  File "/home/biqu/klipper/klippy/configfile.py", line 273, in _build_config_wrapper
    self._parse_config(data, filename, fileconfig, set())
  File "/home/biqu/klipper/klippy/configfile.py", line 261, in _parse_config
    self._resolve_include(filename, include_spec, fileconfig,
  File "/home/biqu/klipper/klippy/configfile.py", line 234, in _resolve_include
    raise error("Include file '%s' does not exist" % (include_glob,))
configparser.Error: Include file '/home/biqu/printer_data/config/generic-bigtreetech-xxx.cfg' does not exist
webhooks client 281473427744368: New connection
webhooks client 281473427744368: Client info {'program': 'Moonraker', 'version': 'v0.8.0-324-ga3e4dac'}
